### ISSUE TITLE
mass failure when using molden

### ISSUE THREAD

**Issue by jesusG-V** [nubakery/bagel#106]
Hello,

It looks that there is a problem when using a MOLDEN file to calculate the frequencies, the mass of the atoms are lost in the reading of the MOLDEN file, any solution?

Attached I put an example, using regular input (rimp2.json), starting from a molden file (rimp2_molden.json) and starting from a save_ref file (rimp2_archive.json)
[hess_check.zip](https://github.com/nubakery/bagel/files/1350386/hess_check.zip)

**Comment by shiozaki** [comment]
@bessvlai 
Hi Bess, Do you know what's going wrong? Did you write a code that accepts specification of masses from the Molden file? Is there any way to get around it?

**Comment by bessvlai** [comment]
I haven’t implemented reading the masses from a molden file. I will take a look at it tonight regarding how to get around it.

**Comment by shiozaki** [comment]
@bessvlai if there is no easy way, and you find the need of a bit of coding, let me know. I am in the code right now and have time to write some.

**Comment by shiozaki** [comment]
@jesusG-V Hold on, I will fix this by this weekend (it's trivial, but needs a bit of coding).

**Comment by jesusG-V** [comment]
@shiozaki no problem, I am managing without it, just thought that it will good for you to know

**Comment by shiozaki** [comment]
Sorry for being very very slow, but this has been fixed in the commit 1981a4a83bdd13e3c34bc0aada7c59084946fd11 (current as of now in the master branch).

This commit also allows users to specify the masses in the molecule block.
```
{ "atom" : "C", "xyz" : [   0.0,  0.0,  0.1 ],  "mass" : 12.0 },
```
This information, however, won't be transferred through the Molden interface, because the Molden format does not specify the syntax for the atomic masses.
